_posts/2015-06-17-Estonia.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "What Estonia can teach us about the future of government records"
+date: 2015-06-17
+layout: post
+author:
+- emileigh
+tags:
+- usds
+- event
```

Comment:
sorry, one more: this should be `events` maybe also add a tag for "estonia" or some way of capturing international collaboration?